src/main/java/cyrus/utility/DateUtility.java

```diff
@@ -0,0 +1,48 @@
+package cyrus.utility;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Utility to deal with {@code LocalDate}.
+ */
+public class DateUtility {
```

Comment:
I like how you have abstracted most of your code into distinct objects that is easy to read and understand!